wiz-admission-controller/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for wiz-admission-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+wizApiToken:
+  clientId: ""
+  clientToken: ""
+  env: "prod" # (prod/ gov)
+
+replicaCount: 2
+
+image:
+  repository: docker.io/library/wiz-admission-controller-server
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose defalt is the chart appVersion.
+  tag: "latest"
```

Comment:
Probably better not to publish a chart that consumes always latest as default.

Reply:
Add todo after we publish our first version